cmd/argoexec/commands/emissary_test.go

```diff
@@ -49,7 +49,7 @@ func TestEmissary(t *testing.T) {
 	})
 	t.Run("Sub-process", func(t *testing.T) {
 		_ = os.Remove(varRunArgo + "/ctr/main/stdout")
-		err := run("(sleep 60; echo 'should not wait for sub-process')& echo -n hello")
+		err := run(`(sleep 60; echo 'should not wait for sub-process')& echo "hello\c"`)
```

Comment:
For other readers, note that `\c` is equivalent to the `-n` flag. They [both mean](https://www.gnu.org/software/bash/manual/bash.html#index-echo) to not output a trailing newline